tool/docs/deps.bzl

```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (C) 2022 Vaticle
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+load("@rules_python//python:pip.bzl", "pip_parse")
```

Comment:
is it worth calling this file `python_deps.bzl` the way you did for `java_docs`?

Reply:
Yes